packages/cli/src/environments/sourceControl/sourceControlExport.service.ee.ts

```diff
@@ -186,6 +190,11 @@ export class SourceControlExportService {
 			const removedFiles = await this.rmDeletedWorkflowsFromExportFolder(sharedWorkflows);
 			// write the workflows to the export folder as json files
 			await this.writeExportableWorkflowsToExportFolder(sharedWorkflows);
+			// write list of owners to file
+			const ownersFileName = this.getOwnersPath();
+			const owners: Record<string, string> = {};
+			sharedWorkflows.forEach((e) => (owners[e.workflowId] = e.user.email));
```

Comment:
One problem I see here is that we're always sending the full file, regardless of the workflows actually exported.

One problem I had is that I have over 100 workflows, but I exported a single one. My owners file contains all the 100 workflow owner names, even though my repo only contains a single workflow file.

I'm thinking that maybe we should read the file contents and append the rows that were added WDYT?